pandas/io/parquet.py

```diff
@@ -25,6 +25,10 @@ def get_engine(engine):
         except ImportError:
             pass
 
+        raise ImportError("unable to find a usable engine\n"
+                          "tried using: pyarrow, fastparquet\n\n"
```

Comment:
why are you changing this? IIRC this is a separeate issue, make it a separate PR